doc/foreign-code.rst

```diff
@@ -257,7 +257,7 @@ descriptions by referencing them as the module specified in optional
    - ``(headers (preamble <preamble>)`` adds directly the preamble. Variables
      can be used in ``<preamble>`` such as ``%{read: }``.
 
-- Since the Dune's ctypes features is still experimental, it could be useful to
+- Since the Dune's ``ctypes`` features is still experimental, it could be useful to
   add additionnal dependencies in order for example to make sure that local
```

Comment:
```suggestion
- Since Dune's ``ctypes`` features are still experimental, it could be useful to
  add additional dependencies in order to make sure local
```